scene/2d/camera_2d.h

```diff
@@ -99,8 +98,6 @@ class Camera2D : public Node2D {
 
 	void _update_process_internal_for_smoothing();
```

Comment:
Unrelated, but this method is unused and can be removed.
Same with `_is_editing_in_editor` above.